changelogs/CHANGELOG-2.md

```diff
@@ -2580,7 +2580,7 @@ Yep. Pretty sure. Maybe. Hmm... I hope.
 
 *Sighs audibly.*
 
-[Let us know](http://github.com/npm/npm/issues/new) if we broke something else
+[Let us know](https://github.com/npm/npm/issues/new) if we broke something else
```

Comment:
Should this be https://github.com/npm/cli/issues/new

Reply:
Probably but not related to the https change :P

Reply:
heh, looks like both links are actually broken.

Reply:
Because the old repo has been archived...

I can fix this one, np.

Reply:
There are thousands of old links that could be updated. The commit ones should be safe to do since the hashes should be the same. PR/issue links should stay. And changelog.js needs to be updated.